feature-libs/product/configurators/common/components/service/configurator-storefront-utils.service.ts

```diff
@@ -105,4 +105,35 @@ export class ConfiguratorStorefrontUtilsService {
       }
     }
   }
+
+  /**
+   * Removes the value ID from the element ID to get the attribute ID.
+   *
+   * @param {string} id - HTML element ID
+   */
+  getAttributeId(id: string): string {
+    const index = id?.lastIndexOf('--');
+    return id?.substring(0, index);
+  }
+
+  /**
+   * Returns true if the focus changed from one attribute to any other element.
+   * Returns false if the focus change occurred inside the same attribute (i.e. from one value to another of the same attribute)
+   *
+   * @param {FocusEvent} event FocusEvent to be checked
+   */
+  attributeLostFocus(event: FocusEvent): boolean {
```

Comment:
Is this method used at all?

Reply:
Done! The method is removed with the corresponding tests